DependencyInjection/Configuration.php

```diff
@@ -125,6 +125,14 @@ public function getConfigTreeBuilder()
                             ->defaultTrue()
                             ->info('Enable automatic registration of annotations with JMSDiExtraBundle')
                         ->end()
+                        ->scalarNode('role_admin')
+                            ->defaultValue('ROLE_SONATA_ADMIN')
+                            ->cannotBeEmpty()
```

Comment:
it will be bc anyways because it is a new key, or am I missing something @greg0ire ?